src/components/input/input.jsx

```diff
@@ -50,6 +51,10 @@ Input.propTypes = {
   cssLabel: PropTypes.string,
   disabled: PropTypes.bool,
   id: PropTypes.string.isRequired,
+  inputRef: PropTypes.oneOfType([
```

Comment:
I think our React version in our Mavenlink application supports the latest ref types. If that is the case, I would like to avoid supporting the function-based refs to have more reason to move our codebase forward:

In addition, a "ref" is actually meant to just be a simple pointer -- the way we use it implies HTML elements already. This is actually a bit more universal (think React Native in the future) and acceptable:

```
inputRef: PropTypes.shape({ current: PropTypes.any }),
```

Reply:
Change made. 
We were thinking of being specific towards the input html element since that's what it was being used for, and not expecting a reference to another element type.